README.md

```diff
@@ -1,7 +1,132 @@
-# Mattermost Boards (Focalboard Plugin)
+# Mattermost Boards
+
+### A self-hosted plugin for managing boards withing a Mattermost installation
+
+Mattermost boards plugins is an open source, multilingual, self-hosted project management tool that's an alternative to Trello, Notion, and Asana.
 
 **[Mattermost Boards](https://mattermost.com/boards/)** is the Mattermost plugin version of Focalboard that combines project management tools with messaging and collaboration for teams of all sizes. To access and use **Mattermost Boards**, install or upgrade to Mattermost v6.0 or later as a [self-hosted server](https://docs.mattermost.com/guides/deployment.html?utm_source=focalboard&utm_campaign=focalboard) or [Cloud server](https://mattermost.com/get-started/?utm_source=focalboard&utm_campaign=focalboard). After logging into Mattermost, select the menu in the top left corner of Mattermost and select **Boards**.
```

Comment:
Is the ``Mattermost v6.0 or later`` reference here correct?

Reply:
Focalboard references; all Boards docs are no longer on the product docs site, and have moved to the GH repo

Reply:
Also, this paragraph starts with `Boards is ...` and the one above as well. Should we keep just one definition of what Boards plugin is to not repeat it ?

Reply:
Yes, please!

Reply:
>  To access and use **Mattermost Boards**, install or upgrade to Mattermost v6.0 or later as a [self-hosted server](https://docs.mattermost.com/guides/deployment.html?utm_source=focalboard&utm_campaign=focalboard) or [Cloud server](https://mattermost.com/get-started/?utm_source=focalboard&utm_campaign=focalboard). 

This is no longer accurate as Boards is no longer installed or enabled by default and also not available in Cloud. We need to change this to say they need to upload the plugin.

Reply:
As of now will remove the complete para, if needed we can added later.